src/Utils/viewport.ts

```diff
@@ -0,0 +1,9 @@
+export const getViewportWidth = () => {
```

Comment:
I may make more sense to do

```typescript
const getViewport: () => { width: number, height: number } = () => {
 ...
} 
```

usage

```typescript
const { width } = getViewport()
```

that way you can also get the height of the viewport for another use-case